infertrade/algos/community/signals.py

```diff
@@ -296,7 +300,8 @@ def aroon(df: pd.DataFrame, window: int = 25) -> pd.DataFrame:
 
 def rate_of_change(df: pd.DataFrame, window: int = 12) -> pd.DataFrame:
     """
-    Rate of Change is momentum-based technical indicator that measures the percentage change in price between the current price and the price a certain number of periods ago.
+    Rate of Change is momentum-based technical indicator that measures the percentage change in price between the
```

Comment:
All the docstrings appear to have broken in this PR

Reply:
I'll take a look and fix it as soon as I can